docs/source/security/authentication.mdx

```diff
@@ -97,13 +110,13 @@ For our first example, let's look at a resolver that's only accessible with a va
 
```

Comment:
@glasser I noticed you reopened an issue about handling thrown errors in context, is L101 (above) possibly affected / should it get a TODO(AS4)?

Reply:
That's why I thought about it, yeah. Though hopefully the answer will be "what is written here just works".

Reply:
I'll add a TODO here just in case!